deepchecks/nlp/checks/model_evaluation/prediction_drift.py

```diff
@@ -143,28 +144,31 @@ def run_logic(self, context: Context) -> CheckResult:
             value: drift score.
             display: prediction distribution graph, comparing the train and test distributions.
         """
-        context.raise_if_token_classification_task(self)
-
         train_dataset = context.train.sample(self.n_samples, random_state=context.random_state)
         test_dataset = context.test.sample(self.n_samples, random_state=context.random_state)
         model = context.model
 
-        # Flag for computing drift on the probabilities rather than the predicted labels
-        proba_drift = ((len(context.model_classes) == 2) and (self.drift_mode == 'auto')) or \
-                      (self.drift_mode == 'proba')
-        model_classes = context.model_classes
-
-        if proba_drift:
-            if context.is_multi_label_task():
-                raise DeepchecksValueError('Cannot use proba drift mode for multi-label tasks')
-            train_prediction = np.array(model.predict_proba(train_dataset))
-            test_prediction = np.array(model.predict_proba(test_dataset))
-        elif context.is_multi_label_task():
-            train_prediction = convert_multi_label_to_multi_class(model.predict(train_dataset), model_classes)
-            test_prediction = convert_multi_label_to_multi_class(model.predict(test_dataset), model_classes)
+        if context.task_type == TaskType.TOKEN_CLASSIFICATION:
+            train_prediction = np.array(list(itertools.chain(*model.predict(train_dataset)))).reshape(-1, 1)
```

Comment:
i think prediction are np arrays (need to verify for token classification) so the itertools may be redundant

Reply:
do the same for label drift :)

Reply:
OOps